rotkehlchen/chain/ethereum/utils.py

```diff
@@ -208,11 +212,25 @@ def try_download_ens_avatar(
     if avatar_url == '':
         return  # Avatar is not set
 
-    try:
-        avatar = requests.get(avatar_url, timeout=DEFAULT_TIMEOUT_TUPLE).content
-    except RequestException as e:
-        log.error(f'Got error {str(e)} during querying ens avatar for {ens_name}.')
-        return
+    avatar = None
+    if avatar_url.startswith('eip155'):  # an NFT is set. Let's try first ENS app's own metadata
```

Comment:
We have `EVM_CHAIN_DIRECTIVE`, maybe use it?

```suggestion
    if avatar_url.startswith(EVM_CHAIN_DIRECTIVE):  # an NFT is set. Let's try first ENS app's own metadata
```